build.gradle

```diff
@@ -12,21 +14,39 @@ if ((buildTag == null || buildTag.isEmpty()) && !version.endsWith('-SNAPSHOT'))
     version += '-SNAPSHOT'
 }
 
-if (buildAccess == 'internal') {
-    group = group.replace('org.openmbee', 'gov.nasa.jpl.cae')
+def profiles = new Properties()
+def parser = new JsonSlurper()
+def defaults = new Properties()
+file("buildProfiles/opensource.properties").withInputStream { defaults.load(it) }
+
+if (buildProfile != 'opensource') {
+    try {
+        file("buildProfiles/" + buildProfile + ".properties").withInputStream { profiles.load(it) }
+    } catch (FileNotFoundException e) {
```

Comment:
Suppressing errors / misconfiguration make them harder to debug, especially in a noisy log. Best to fail early.

Reply:
The purpose of that suppression is to handle the default case (i.e. if not specified set build profile to opensource.

Reply:
I think I now handle that in a different way so 🤷